linkstone-annotations/src/main/java/net/glowstone/linkstone/annotations/LConstructor.java

```diff
@@ -0,0 +1,15 @@
+package net.glowstone.linkstone.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Tell the compiler in which version the annotated constructor exists.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.CONSTRUCTOR)
+public @interface LConstructor {
+    Version[] version();
+}
```

Comment:
Why not just use `@LClassfile` for this as well?

Reply:
If we have an annotation for Packages, Classes, Methods and Fields, why should we not also have an annotation for Constructors?

Another issue is that the Classfile annotation may take a name for obfuscation while constructors don't have names.